python/python/Ice/PythonDispatcher.py

```diff
@@ -0,0 +1,92 @@
+# Copyright (c) ZeroC, Inc.
```

Comment:
I would rather not "dispatcher" for this Util file. And we should prefix Python code with Python.

What about simply Dispatch.py?